src/pages/TermsOfService.test.ts

```diff
@@ -115,4 +147,43 @@ describe('TermsOfService', () => {
     const acceptButton = screen.queryByText('Accept');
     expect(acceptButton).not.toBeInTheDocument();
   });
+  it('calls the acceptTos endpoint when the accept tos button is clicked', async () => {
+    // Arrange
+    const termsOfService = {
+      userHasAcceptedLatestTos: false,
+      permitsSystemUsage: true,
+    };
+
+    const { acceptTosFn } = setupMockAjax(termsOfService);
+
+    // Act
+    await act(async () => { render(h(TermsOfServicePage)) }) //eslint-disable-line
+
+    // Assert
+    const acceptButton = screen.getByText('Accept');
+    await act(async () => fireEvent.click(acceptButton));
+    expect(acceptTosFn).toHaveBeenCalled();
+  });
+
+  it('calls the rejectTos endpoint when the reject tos button is clicked', async () => {
+    // Arrange
+    const termsOfService = {
+      userHasAcceptedLatestTos: false,
+      permitsSystemUsage: false,
+    };
+
+    const { rejectTosFn } = setupMockAjax(termsOfService);
+
+    // Act
+    await act(async () => { render(h(TermsOfServicePage)) }) //eslint-disable-line
+
+    // Assert
+    const rejectButton = screen.queryByText('Decline and Sign Out');
+    expect(rejectButton).toBeInTheDocument();
+    expect(rejectButton).not.toBeNull();
+    if (rejectButton) {
+      await act(async () => rejectButton.click());
+    }
```

Comment:
This could be simplified similar to the other test case.

```suggestion
    const rejectButton = screen.getByText('Decline and Sign Out');
    await act(async () => fireEvent.click(rejectButton));
```

Reply:
oh duh thank you, was being lazy 🤦